source/chapters/library.rst

```diff
@@ -475,6 +474,28 @@ Mixxx supports the following filters:
      You can put a space after the colon but currently there must be no space
      between the operator and the number.
 
+In addition to combining filters directly (effectively an "AND"), you can also combine queries using the "OR" operator, spelled ``|``.
+
+Examples
+  ::
+  
+    genre:house | genre:techno
+
+  The "OR" operator has a lower precedence than regular filters, so you can create larger queries too. The following query, for example, would match all tracks that are 5-star Hip-Hop or Pop:
+
+  ::
+
+    genre:hip-hop rating:5 | genre:pop
+
+  Similar to popular search engines, the "OR" operator also supports the alternative spelling ``OR``, so the previous query could equivalently be written as:
+
+  ::
+
+    genre:hip-hop rating:>=4 OR genre:pop
+
+.. note::
+  The spelled-out "OR" operator is case-sensitive, i.e. searching for ``or`` will search for the literal string ``or``. Using the quoted form ``"OR"`` will also search literally.
+
```

Comment:
I feel that, in this place, this important section does not get the attention it deserves.
What do you think about moving it up, below the first paragraph?

> ### Using search operators
> 
>Search operators allow you to form more complex search queries. They allow you to limit certain search terms to particular properties of your tracks.

<--

> Mixxx supports the following filters:

Reply:
The failing link check is unrelated (we need to merge 2.3 into 2.4, 2.4 into main) and pre-commit only found a trailing whitespace.

Reply:
My idea was to first introduce the "OR" operator itself, then the spelling and finally discuss this rather unusual (but not inconceivable) edge case/gotcha. Note that strings only containing "OR" are unaffected, so I would expect such queries to be pretty uncommon. Users searching in lowercase would never run into this anyway.

Also I think it would be nice to keep everything about the "OR" operator in a single section.

Could you elaborate on why this note deserves more attention?

Reply:
> Also I think it would be nice to keep everything about the "OR" operator in a single section.

Of course. I think this was just a misunderstanding, by 'section' I mean the entire new OR section.
Anyway, the placement is fine for now, maybe it's just that I find the current stylesheet a bit inconclusive, so it's a general issue IMHO. With the bullet points it looks like OR belongs **Special filtering**. 
Maybe it would help if OR gets a headline **Combine filter with OR**.
Later on we may move some AND examples from **Special filtering** to **Combine filters** where we clarify the default AND and explicit OR. But that's out of scope here.

Reply:
Ah sorry, yeah, that was a misunderstanding on my part. That sounds good!